src/blocks/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
```

Comment:
У тебя же это правило прописано в style.css))
```suggestion
  box-sizing: border-box;
```

Reply:
Delete it from here